tasties_app/migrations/0004_merge_20221229_1649.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 4.1.3 on 2022-12-29 16:49
```

Comment:
Please, please get rid of this file. There's no need to auto merge if we state our dependencies properly in the migrations themselves.